docs/source/getting-started/quickstart/template-root/daml/Main.daml

```diff
@@ -7,6 +7,7 @@ module Main where
 import Iou
 import IouTrade()
 
+setup : Scenario IouCid
```

Comment:
Do we not want to delete this whole thing now?

Reply:
It is still being used in the documentation for describing scenarios.